yb-voyager/cmd/constants.go

```diff
@@ -58,29 +58,38 @@ const (
 	ROW_UPDATE_STATUS_COMPLETED     = 3
 	COLOCATION_CLAUSE               = "colocation"
 	//phase names used in call-home payload
-	ANALYZE_PHASE          = "analyze-schema"
-	EXPORT_SCHEMA_PHASE    = "export-schema"
-	EXPORT_DATA_PHASE      = "export-data"
-	IMPORT_SCHEMA_PHASE    = "import-schema"
-	IMPORT_DATA_PHASE      = "import-data"
-	END_MIGRATION_PHASE    = "end-migration"
-	ASSESS_MIGRATION_PHASE = "assess-migration"
-	IMPORT_DATA_FILE_PHASE = "import-data-file"
+	ANALYZE_PHASE                    = "analyze-schema"
+	EXPORT_SCHEMA_PHASE              = "export-schema"
+	EXPORT_DATA_PHASE                = "export-data"
+	EXPORT_DATA_FROM_TARGET_PHASE    = "export-data-from-target"
+	IMPORT_SCHEMA_PHASE              = "import-schema"
+	IMPORT_DATA_PHASE                = "import-data"
+	IMPORT_DATA_SOURCE_REPLICA_PHASE = "import-data-to-source-repilca"
+	IMPORT_DATA_SOURCE_PHASE         = "import-data-to-source"
+	END_MIGRATION_PHASE              = "end-migration"
+	ASSESS_MIGRATION_PHASE           = "assess-migration"
+	IMPORT_DATA_FILE_PHASE           = "import-data-file"
 	//...more phases
 	OFFLINE        = "offline"
 	LIVE_MIGRATION = "live migration"
 	BULK_DATA_LOAD = "bulk data load from flat files"
 
+	LIVE_MIGRATION_WITH_FALL_FORWARD = "live migration with fall-forward"
```

Comment:
since these are essentially enums, it would be good to not have spaces. Let's use underscores. Also for `LIVE_MIGRATION`

Reply:
Yeah, that is better and will be consistent with other enums by keeping hyphens instead of spaces.